fetch.bs

```diff
@@ -4820,17 +4968,30 @@ these steps:
 
         <ol>
          <li><p>Let <var>bytes</var> be the transmitted bytes.
+         <li><p>Let <var>timingInfo</var> be the <var>fetchParams</var>'s <a for="fetch params">timing info</a>.
 
          <li><p>Let <var>codings</var> be the result of <a>extracting header list values</a> given
          `<code>Content-Encoding</code>` and <var>response</var>'s <a for=response>header list</a>.
 
+         <li><p>Set <var>timingInfo</var>'s <a for="fetch timing info">transfer size</a> to the result of
+         adding <var>timingInfo</var>'s <a for="fetch timing info">transfer size</a> and <var>bytes</var>.
+
+         <p class=XXX>TODO: <a for="fetch timing info">transfer size</a> should also include the total size of the headers.
```

Comment:
I think there's no need to bother with that, due to https://github.com/w3c/resource-timing/issues/238
In the latest WG discussion, we talked about just adding a fixed amount of bytes as headers, to avoid exposing sensitive information, while keeping backwards compat.

Reply:
I think, then, that for now fetch should not have `transferSize` at all, and RT can bother with adding a fixed amount to `encodedBodySize`.